src/js/profile/mobile/widget/Chip.js

```diff
@@ -265,8 +259,9 @@
 				ui.button = ui.button || element.querySelector(selectors.button);
 
 				ns.widget.Button(ui.button, {
-					icon: self._getButtonIcon()
+					icon: self.options.icon
 				});
+				this._setButtonColor(ui.button);
```

Comment:
_setButtonColor seems that doesn't have a parameter.
Is it correct?

Reply:
Thanks for review. This is mistake and method name also make confusion. I'll change this.